packager/packager.cc

```diff
@@ -1328,6 +1375,16 @@ class ImportTreeBuilder final {
                 continue;
             }
 
+            if (package.layer() < importedPackage.layer()) {
```

Comment:
Trying to remember the stripe logic, don't we only check this for `import` not `test_import`?

Also should have a test case for that.